packages/Webkul/Customer/src/Captcha.php

```diff
@@ -146,8 +146,8 @@ public function getValidationMessages($messages = []): array
     {
         return $this->isActive()
             ? array_merge($messages, [
-                'g-recaptcha-response.required' => __('customer::app.admin.system.captcha.validations.required'),
-                'g-recaptcha-response.captcha' => __('customer::app.admin.system.captcha.validations.captcha')
+                'g-recaptcha-response.required' => __('customer::app.validations.captcha.required'),
+                'g-recaptcha-response.captcha' => __('customer::app.validations.captcha.captcha')
```

Comment:
Please modify the "trans" method on the place of ( __ ).

Reply:
Replace (_) into trans method